testsuite/classlibrary/TestFunction.sc

```diff
@@ -66,6 +72,8 @@ CommonTestClass {
 		}
 	}
 
+	*reset { x = nil; y = 7; }
+	reset { a = nil; b = 3; }
```

Comment:
wouldn't these make more sense as a `tearDown` in `TestFunction`?

Reply:
Well, not really in this case - the reset happens in CommonTestClass, and setUp und tearDown would be in TestFunction. 
So TestFunction:setUp would need to call CommonTestClass:reset, and create the CommonTestClass instance, and call reset on that. 
Not sure that that is cleaner or easier to understand.
Maybe I dont get how you advised to implement that?
best a

Reply:
ope, originally i thought TestFunction inherited from UnitTest. now i have additional questions:

- why call `reset` on `TestFunction`? `reset` is only implemented on UnitTest as a class method, but you're calling it on `this` from within an instance method, so AFAICT that method call will invoke `Object.reset`, which is a nop.
- why add an instance method for `reset` on `CommonTestClass` when it is never used?

Reply:
done in bcebca9 I hope:
class method reset is now Meta_CommonTestClass:initClass (which is what it is), 
and TestFunction:setup calls it so the class variables are reset before every run of a TestFunction test.